### ISSUE TITLE
Smoke Test Failure: sidebar has no accessibility violations

### ISSUE THREAD

**Issue by alexdima** [microsoft/vscode#321143]
https://dev.azure.com/monacotools/Monaco/_build/results?buildId=447184&view=logs&j=e913b328-1258-5362-123b-17c269eb45c3&t=385e183f-c1be-5a14-ec9a-230effadb445&l=171

```
  1) VSCode Smoke Tests (Electron)
       Accessibility
         Workbench
           sidebar has no accessibility violations:
     Error: Accessibility violations found:

[aria-valid-attr-value] ARIA attributes must conform to valid values (critical)
  Help URL: https://dequeuniversity.com/rules/axe/4.11/aria-valid-attr-value?application=axeAPI
  Element: #list_id_3_0
    Class: monaco-list-row
    HTML: <div class="monaco-list-row" role="treeitem" data-index="0" data-last-element="true" data-parity="even" aria-setsize="1" aria-posinset="1" id="list_id_3_0" aria-selected="false" aria-label="vscode-smoketest-exp..." aria-level="0" draggable="true" style="top: 0px; height: 22...">
    Issue: Fix all of the following:
  Invalid ARIA attribute value: aria-level="0"

Total: 1 violation(s) affecting 1 element(s)
      at Proxy.assertNoAccessibilityViolations (D:\a\_work\1\s\test\automation\out\playwrightDriver.js:700:19)
      at async Context.<anonymous> (out\areas\accessibility\accessibility.test.js:47:17)

```
